### PR TITLE
chore: Add unit tests for formatted balance

### PR DESCRIPTION
**Pull request by holzeis**

### CODE REVIEW

File: mobile/lib/features/wallet/application/util.dart [hunk 1]
```diff
@@ -3,3 +3,37 @@ String truncateWithEllipsis(int cutoff, String text) {
       ? text
       : '${text.substring(0, (cutoff / 2).floor())}...${text.substring(text.length - (cutoff / 2).ceil(), text.length)}';
 }
+
+// Formats any number less than 10 bitcoin in the following format
```
Comment (bonomat) [inline mobile/lib/features/wallet/application/util.dart:7]: I believe doc comments have 3 `/`.

**Review by bonomat — APPROVED** [review]
Nice

File: mobile/lib/features/wallet/application/util.dart (hunk 1 above)
Comment (holzeis) [reply mobile/lib/features/wallet/application/util.dart:7]: Also in dart? 🤔

Comment (bonomat) [reply mobile/lib/features/wallet/application/util.dart:7]: Yap :) https://dart.dev/effective-dart/documentation

Comment (holzeis) [reply mobile/lib/features/wallet/application/util.dart:7]: Next time then 🙂

**Pull request merged**